### PR TITLE
🚀 New `/solutions/collaboration` page

### LINKED ISSUE #3437: Create new landing page for /solutions/collaboration
You can find the mockup [here](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/website?node-id=9326%3A21637&t=3p5d2IB2WlYyBaZe-1)

Reviewer: @MichaelAring

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
## Description

Review on [`/solutions/collaboration`](https://deploy-preview-3448--gitpod-kumquat.netlify.app/solutions/collaboration)

## Related Issue(s)

Fixes #3437

## Release Notes

```release-note
Website: Gitpod Collaboration Solutions page
```

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3448"></a>

### CODE REVIEW

File: src/routes/solutions/collaboration.svelte [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+<OpenGraph
+  data={{
+    description:
+      "Gitpod gives a perfect development environment to everyone on the team.",
+    title: "Solution: Collaboration - Gitpod",
+    keywords:
+      "cloud dev environment, cde, remote development, developer experience, collaboration, solution, solutions",
+  }}
+/>
```
Comment (Siddhant-K-code) [inline src/routes/solutions/collaboration.svelte:26]: @MichaelAring Feel free to suggest some other Meta Descriptions & Title page.

File: src/routes/solutions/collaboration.svelte [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    <p class="text-large max-w-xl">
+      Gitpod gives a perfect development environment to everyone on the team.
+    </p>
+  </div>
+  <Quote
+    quote="
+    With Gitpod, I can review any PR in a coding environment where I can edit, build and test the contribution, by just prepending <span class='text-dark-grey inline-block px-micro mt-macro bg-tertiary rounded-xl'><code>gitpod.io/#</code></span>"
+    author={{
+      // TODO: Update this to a real name
+      name: "xyz",
```
Comment (Siddhant-K-code) [inline src/routes/solutions/collaboration.svelte:44]: @MichaelAring We need a real name to mention here. [_Internal conversastion_](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj?node-id=9326:21637#370028817)

Comment (MichaelAring) [reply src/routes/solutions/collaboration.svelte:44]: Johannes Schickling

Comment (MichaelAring) [inline src/routes/solutions/collaboration.svelte]:
```suggestion
      name: "Johannes Schickling",
```

File: src/lib/components/explore.svelte [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
   const {
     title = "Get Started <span>for&nbsp;Free</span>",
-    description = "Spin up fresh cloud development environments for each task, in the cloud, in seconds.",
+    description = "Spin up fresh, automated dev environments for each task, in the cloud, in seconds - no contracts or banking details required.",
```
Comment (MichaelAring) [inline src/lib/components/explore.svelte:12]:
```suggestion
    description = "Spin up fresh, automated dev environments for each task, in the cloud, in seconds - no credit card required.",
```

**Review by MichaelAring — CHANGES_REQUESTED** [review]
editorial changes

**Comment by MichaelAring** [comment]
Please test new Quizlet customer story link before merging.

File: src/routes/solutions/collaboration.svelte [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  import Explore from "$lib/components/explore.svelte";
+</script>
+
+<OpenGraph
+  data={{
+    description:
+      "Gitpod gives a perfect development environment to everyone on the team.",
+    title: "Solution: Collaboration - Gitpod",
+    keywords:
+      "cloud dev environment, cde, remote development, developer experience, collaboration, solution, solutions",
```
Comment (MichaelAring) [inline src/routes/solutions/collaboration.svelte:24]:
```suggestion
      "cloud development environment, cde, remote development, developer experience, collaboration, snapshot, workspace sharing",
```

**Review by MichaelAring — CHANGES_REQUESTED** [review]
meta tags change

**Review by MichaelAring — APPROVED** [review]
looks good!

**Pull request merged**